### PR TITLE
Add RGB value support (#43)

### LINKED ISSUE #43: RGB support
**Is your feature request related to a problem? Please describe.**
Since the 1.16 version Minecraft has implemented the possibility to use RGB codes in messages, actually BedWars1058 not supports this feature.

**Describe the solution you'd like**
Implementing a config option like **rgb/hex-enable:true/false** and adding RGB support (mabye by using some libraries like IridiumColorAPI)

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by Xxyuri2005xX**
- implement RGB support via IridiumcolorAPI
- option for enabling RGB in messages

closes #43

### CODE REVIEW

File: bedwars-plugin/src/main/java/com/andrei1058/bedwars/language/English.java [hunk 1]
```diff
@@ -40,6 +40,7 @@ public English() {
 
         YamlConfiguration yml = getYml();
         yml.options().copyDefaults(true);
+        yml.options().header("RGB are supported with 3 methods: <SOLID:FF0080> -> Solid, <RAINBOW1></RAINBOW> -> Rainbow (number customizable) & <GRADIENT:2C08BA></GRADIENT:028A97> -> Gradient");
```
Comment (tomkeuper) [inline bedwars-plugin/src/main/java/com/andrei1058/bedwars/language/English.java]: this sounds better to me: 'RGB is supported'

File: bedwars-api/src/main/java/com/andrei1058/bedwars/api/language/Language.java [hunk 2]
```diff
@@ -95,8 +96,11 @@ public String getLangName() {
     public static String getMsg(Player p, String path) {
         if (p == null) return getDefaultLanguage().m(path);
         BedWars api = Bukkit.getServicesManager().getRegistration(BedWars.class).getProvider();
-        return ChatColor.translateAlternateColorCodes('&', api.getSupportPapi().replace(p, langByPlayer.getOrDefault(p.getUniqueId(), getDefaultLanguage()).m(path).replace("{prefix}", (prefixStatic == null ? "" : prefixStatic))));
+        //return ChatColor.translateAlternateColorCodes('&', api.getSupportPapi().replace(p, langByPlayer.getOrDefault(p.getUniqueId(), getDefaultLanguage()).m(path).replace("{prefix}", (prefixStatic == null ? "" : prefixStatic))));
 //        return langByPlayer.getOrDefault(p.getUniqueId(), getDefaultLanguage()).m(path).replace("{prefix}", (prefixStatic == null ? "" : prefixStatic));
+
```
Comment (tomkeuper) [inline bedwars-api/src/main/java/com/andrei1058/bedwars/api/language/Language.java:99]: Old code can be removed

**Comment by Xxyuri2005xX** [comment]
fixing

File: bedwars-plugin/pom.xml [hunk 3]
```diff
@@ -311,6 +311,14 @@
                             <pattern>com.andrei1058.spigot.sidebarutils</pattern>
                             <shadedPattern>com.andrei1058.bedwars.libs.updater</shadedPattern>
                         </relocation>
+                            <relocation>
```
Comment (tomkeuper) [inline bedwars-plugin/pom.xml]: extra indent? O-o

**Review by tomkeuper — APPROVED** [review]

File: bedwars-plugin/pom.xml (hunk 3 above)
Comment (Xxyuri2005xX) [reply bedwars-plugin/pom.xml]: time a few minutes and i check

Comment (Xxyuri2005xX) [reply bedwars-plugin/pom.xml]: fixed

**Comment by tomkeuper** [comment]
Just a thought, do we even need an option to disable it? Because all that will do is use basically the same method just without the rgb formatting right?

**Comment by Xxyuri2005xX** [comment]
I don't know about the option, from my point of view it might come in handy to keep it

**Review by porudev — COMMENTED** [review]
Seems solid to me, but will these effect version without RGB support?

**Comment by tomkeuper** [comment]
> Seems solid to me, but will these effect version without RGB support?

I saw on their repo it will just change to the nearest valid color code

**Review by porudev — APPROVED** [review]

**Pull request merged**